src/status_im2/contexts/wallet/account/view.cljs

```diff
@@ -1,15 +1,66 @@
 (ns status-im2.contexts.wallet.account.view
   (:require [react-native.core :as rn]
             [quo2.core :as quo]
-            [utils.re-frame :as rf]))
+            [react-native.safe-area :as safe-area]
+            [reagent.core :as reagent]
+            [status-im2.contexts.wallet.common.tabs.view :as common]
+            [status-im2.contexts.wallet.common.temp :as temp]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]
+            [status-im2.contexts.wallet.account.style :as style]))
+
+(def tabs-data
+  [{:id :assets :label (i18n/label :t/assets) :accessibility-label :assets-tab}
+   {:id :collectibles :label (i18n/label :t/collectibles) :accessibility-label :collectibles-tab}
+   {:id :activity :label (i18n/label :t/activity) :accessibility-label :activity-tab}
+   {:id :permissions :label (i18n/label :t/permissions) :accessibility-label :permissions}
+   {:id :dapps :label (i18n/label :t/dapps) :accessibility-label :dapps}
+   {:id :about :label (i18n/label :t/about) :accessibility-label :about}])
+
 
 (defn view
   []
-  [rn/view
-   {:style {:flex            1
-            :align-items     :center
-            :justify-content :center}}
-   [quo/text {} "ACCOUNTS PAGE"]
-   [quo/divider-label]
-   [quo/button {:on-press #(rf/dispatch [:navigate-back])}
-    "NAVIGATE BACK"]])
+  (let [top          (safe-area/get-top)
+        selected-tab (reagent/atom (:id (first tabs-data)))]
+    (fn []
+      [rn/view
+       {:style {:flex       1
+                :margin-top top}}
+       [quo/page-nav
+        {:align-mid?            true
+         :mid-section           {:type :text-only :main-text ""}
+         :left-section          {:type     :grey
+                                 :icon     :i/close
+                                 :on-press #(rf/dispatch [:navigate-back])}
+         :right-section-buttons [{:type     :grey
+                                  :label    "[WIP]"
+                                  :on-press #(rf/dispatch [:open-modal :how-to-pair])}]}]
+       [quo/account-overview temp/account-overview-state]
+       [quo/wallet-graph {:time-frame :empty}]
+       [quo/wallet-ctas]
+       [quo/tabs
```

Comment:
should we make a connected component for these wallet tabs? it will make this file a lot easier to read imo 👍

Reply:
I guess you could move this to status-im2.contexts.wallet.common.tabs.view ?

Reply:
Sounds good 👌

Reply:
is there updates to come here? if it is still this 
```
[quo/tabs
{:style          style/tabs
...
         :permissions  [quo/empty-state
                        {:title        (i18n/label :t/no-permissions)
                         :description  (i18n/label :t/no-collectibles-description)
                         :placeholder? true}]
         :dapps        [quo/empty-state
                        {:title        (i18n/label :t/no-dapps)
                         :description  (i18n/label :t/no-collectibles-description)
                         :placeholder? true}]
```
I would stiill suggest to consider moving this as a component to it's own namespace and let's keep the `account/view` quite shallow in details.
perhaps: `status-im2.contexts.wallet.account.tabs.view` 🤔

Reply:
@J-Son89 done!